src/_posts/databases/2000-01-01-maintenance-windows.md

```diff
@@ -0,0 +1,236 @@
+databases. On the customer's side, it mainly consists in an 8 hours timespan
+during which database maintenance operations can be scheduled. The goal of
+maintenance windows is to minimize the impact of maintenance operations on your
+application, while facilitating continuous improvement on our side.
+
+{% note %}
+Currently, Scalingo only leverages maintenance windows to force
+[major-upgrades]({% post_url databases/postgresql/2000-01-01-upgrading %})
+of databases when very specific conditions are met, such as an EOL version
+still running, or critical security issues.
+{% endnote %}
+
+A default 8 hours maintenance window is automatically assigned to all newly
+provisioned database. This maintenance window is set to a random weekday,
+from 9PM to 5AM UTC.
+
+If the given timeframe doesn't align with your business requirements, please
+[adjust it as needed](#configuring-a-database-maintenance-window). We strongly
+advise to schedule the maintenance window when your app is less prone to high
+traffic (e.g. the night between Sunday and Monday) to minimize undesired
+consequences such as a possible downtime.
+
+{% note %}
+Database Maintenance Windows are excluded from the SLA calculation.
+{% endnote %}
+
+
+## Viewing Database Maintenance Windows
+
+### Using the Database Dashboard
+
+1. From your web browser, [open your database dashboard]({% post_url databases/postgresql/2000-01-01-getting-started %}#accessing-the-scalingo-for-postgresql-dashboard)
+2. Select the **Settings** tab.
+3. In the **Settings** submenu, select **Maintenance**
+4. The current Maintenance Window is displayed in the **Maintenance Window**
+   block
+
+### Using the Command Line
+
+1. Make sure you have correctly [setup the Scalingo command line tool]({% post_url platform/cli/2000-01-01-start %})
+2. From the command line, run the following command to view the maintenance
+   window settings:
+   ```bash
+   scalingo --app my-app addons-info <database_type>
+   ```
+   With:
+   - `database_type`: must be either `postgresql`, `mysql`, `redis`, `mongodb`,
+     `elasticsearch` or `influxdb`, depending on the database addon
+
+   The output should look like this:
+   ```text
+   +------------------------+------------------------------+
+   | Database Type          | postgresql                   |
+   | Version                | 13.11.0-1                    |
+   | Status                 | running                      |
+   | Plan                   | postgresql-starter-512       |
+   | Force TLS              | disabled                     |
+   | Internet Accessibility | disabled                     |
+   | Maintenance window     | Tuesdays at 22:00 (08 hours) |
+   +------------------------+------------------------------+
+   ```
+
+
+## Configuring Database Maintenance Windows
+
+Maintenance windows can be configured separately for each database addon. To
+configure it, you'll have to pick a weekday and a start time.
```

Comment:
issue(non-blocking): I don't really like using this kind of contraction in written documents
```suggestion
configure it, you will have to pick a weekday and a start time.
```